### DISCUSSION TITLE
Display firmware for IDEX version

### DISCUSSION THREAD

**Discussion by Schtom** [bigtreetech/BIGTREETECH-TouchScreenFirmware#2009] [General]
Hello,

I would like to put a screen firmware for your TFT screen (I have a TFT50)
A complete firmware FOR the IDEX!

Currently you have IDEX motherboard, but no display firmware for IDEX

Can you help me create a screen firmware specially designed for the IDEX?

I would need to include three commands with three images when selecting the print:

**1st:** _"Normal"_ printing with independent print heads
Command: **M605 S1**

**2nd:** _"Duplication"_ printing with the two heads which prints the same identical part.
Command: **M605 S2**

**3rd:** _"Mirror"_ printing with the two heads which print the same identical part, opposite to each other.
Order: **M605 S2** and **M605 S3**

Can't wait to see if you have a solution to this problem :)
Regards,
Raphael

![idex](https://user-images.githubusercontent.com/4550320/122620358-0bb34500-d093-11eb-9ec5-bdb3b85200b9.jpg)

**Comment by muttstang** [comment]
I'd like to see the same.  I have a TFT35 and am building an IDEX printer.  It would also be great to be able to move the 2nd extruder and do the offsets with the touchscreen
